frontend/openchat-client/src/openchat.ts

```diff
@@ -4512,152 +4513,165 @@ export class OpenChat extends OpenChatAgentWorker {
         }
     }
 
-    private async loadChats() {
-        try {
-            const init = this._liveState.chatsInitialised;
-            chatsLoading.set(!init);
-
-            const updateRegistryTask = !init ? this.updateRegistry() : undefined;
-
-            const chatsResponse = await this.sendRequest({
-                kind: "getUpdates",
-            });
-
-            if (!init || chatsResponse.anyUpdates) {
-                if (updateRegistryTask !== undefined) {
-                    // We need the registry to be loaded before we attempt to render chats / events
-                    await updateRegistryTask;
-                }
+    private async loadChatsResponse(
+        updateRegistryTask: Promise<void> | undefined,
+        init: boolean,
```

Comment:
I know this wasn't changed in this PR but I think calling this `init` is confusing and I've had to check what it means a few times in the past.
It could mean 'this is doing the initialization', but in our case it means 'things are initialized'.
So maybe we should call it `initialized`?

Reply:
Or invert it and call it `initialLoad`?